packages/11ty/_components/figure-image.webc

```diff
@@ -0,0 +1,61 @@
+<template webc:nokeep>
+  <canvas-panel webc:if="isCanvas(id)" @attributes="attributes(id, image_dir).canvasPanel"></canvas-panel>
+  <image-service webc:elseif="isImageService(id)" @attributes="attributes(id, image_dir).imageService"></image-service>
+  <image-sequence webc:elseif="isSequence(id)" @attributes="attributes(id, image_dir).imageSequence"></image-sequence>
+  <image-tag webc:else @attributes="attributes(id, image_dir).imageTag"></image-tag>
+</template>
```

Comment:
Could the template be written using a dynamic webc component, `webc:is="component(id)"`, rather than a webc conditional statements?

Reply:
Apparently, `webc:is` does not support dynamic values; see https://github.com/11ty/webc/issues/143

Reply:
We might add a comment on the implementation quirks, something like
```
<!--- 
  Nota bene: `webc:is` does not support dynamic values (see 11ty/webc#143);
  use webc conditional statements and call the `attributes` function multiple times rather than using a fancier pattern of implementation.
--->
```